cmd/test.go

```diff
@@ -105,7 +105,7 @@ const (
 	// measurement tests
 	testVerdictGood testVerdict = "Good"
 	testVerdictAvg  testVerdict = "Avg"
```

Comment:
```suggestion
	testVerdictAvg  testVerdict = "Okay"
```

Reply:
Let's revise those again when we improve the whole scoring system? Then we can have poor, average, okay, perfect, etc.